include/ensmallen_bits/callbacks/print_loss.hpp

```diff
@@ -0,0 +1,57 @@
+/**
+ * @file print_loss.hpp
+ * @author Marcus Edel
+ *
+ * Implementation of the print loss callback function.
+ *
+ * ensmallen is free software; you may redistribute it and/or modify it under
+ * the terms of the 3-clause BSD license.  You should have received a copy of
+ * the 3-clause BSD license along with ensmallen.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef ENSMALLEN_CALLBACKS_PRINT_LOSS_HPP
+#define ENSMALLEN_CALLBACKS_PRINT_LOSS_HPP
+
+namespace ens {
+
+/**
+ * Print loss function, based on the EndEpoch callback function.
+ */
+class PrintLoss
+{
+ public:
+  /**
+   * Set up the print loss callback class with the width and output stream.
+   *
+   * @param ostream Ostream which receives output from this object.
+   */
+  PrintLoss(std::ostream& output = arma::get_cout_stream()) : output(output)
```

Comment:
`arma::get_cout_stream()` is in Armadillo 8.100 onwards.  See comment for include/ensmallen_bits/callbacks/progress_bar.hpp
https://github.com/mlpack/ensmallen/pull/119/commits/b6edf29b5862decf2ce02d534cf1de1ec7325e21#r290214017